.github/workflows/test.yml

```diff
@@ -30,4 +30,4 @@ jobs:
         working-directory: 'website'
       - uses: untitaker/hyperlink@0.1.27
```

Comment:
Just a question: 

would it be prudent to also update this to the latest `0.1.31`?

The update includes several dependency updates and fixes since January `0.1.27`